libs/batch_environments/docs/index.rst

```diff
@@ -11,3 +11,16 @@
 batch_environments
 ==================
 
+This module allows for the execution of batch jobs, a series of programs
```

Comment:
I would rather word this as "detection of execution as batch jobs". When an HPX program starts this module is essentially responsible for checking if it thinks the program runs as part of a batch job, and if it is to grab settings from the batch job environment (such as the number of nodes).